viz.yaml

```diff
@@ -70,7 +70,7 @@ fetch:
     start.date: "2016-10-5"
     depends: ["sites-data","precip-data"]
     reader: rds
-    refetch: true
+    refetch: FALSE
```

Comment:
Ah...something I might actually be able to contribute. I'm just opening up the Jenkin's build, and it's definitely failing. I'll try to figure out why....